app/experimenter/experiments/tasks.py

```diff
@@ -110,3 +112,56 @@ def add_experiment_comment_task(user_id, experiment_id):
         metrics.incr("add_experiment_comment.failed")
         logger.info("Failed to create bugzilla comment")
         raise e
+
+
+@app.task
+@metrics.timer_decorator("update_experiment_status.timing")
+def update_experiment_status():
+    metrics.incr("update_experiment_status.started")
+    accepted_experiments = Experiment.objects.filter(
+        Q(status=Experiment.STATUS_ACCEPTED) | Q(status=Experiment.STATUS_LIVE)
+    )
+    status_mapping = update_mapping()
+    for experiment in accepted_experiments:
+        try:
+            recipe_data = normandy.get_recipe(experiment.normandy_id)
+            if needs_to_be_updated(recipe_data["enabled"], experiment.status):
+
+                approver_email = recipe_data["approval_request"]["approver"][
+                    "email"
+                ]
+                approver, _ = get_user_model().objects.get_or_create(
+                    email=approver_email
+                )
+
+                experiment.status = status_mapping[experiment.status]
```

Comment:
I guess we should start getting into the habit of using `transaction.atomic()` in places where we do multiple updates:

https://docs.djangoproject.com/en/2.2/topics/db/transactions/#django.db.transaction.atomic